Bio/Align/__init__.py

```diff
@@ -1068,6 +1071,195 @@ def parse_printed_alignment(cls, lines):
         parser.fill(coordinates)
         return sequences, coordinates
 
+    @classmethod
+    def from_alignments_with_same_reference(
```

Comment:
You may want to just call this `from_alignments`, as the other function names also tend to be simple (like `map` and `mapall`).
That said, I don't have a strong opinion on it.

Reply:
I personally think it's better to keep it like this, since it is a self-explaining name. Otherwise you either have to try and fail / read the docs